config/foreman.yml

```diff
@@ -11,7 +11,7 @@
   # Basic Auth:
   #:default_auth_type: 'Basic_Auth'
   :username: 'admin'
-  #:password: 'example'
+  :password: 'changeme'
```

Comment:
You probably do not want to commit this change.

Reply:
Yes, I'd rather not change this :)